### PR TITLE
Upd/Adm with neo-go `v0.99.2`

### LINKED ISSUE #1711: Simplify neofs-adm code after neo-go v0.99.2 update
`actor` package can be useful.
See https://github.com/nspcc-dev/neofs-node/pull/1705#discussion_r949013572

### PR DESCRIPTION
**Pull request by carpawell**
Closes #1711.

### CODE REVIEW

**Review by cthulhu-rider — APPROVED** [review]
Too much changes I'm not familiar with. Code LGTM.

File: cmd/neofs-adm/internal/modules/morph/initialize.go [hunk 1]
```diff
@@ -320,21 +325,28 @@ func (c *clientContext) awaitTx(cmd *cobra.Command) error {
...
+
+func awaitTx(cmd *cobra.Command, c Client, hashes []util.Uint256) error {
 	cmd.Println("Waiting for transactions to persist...")
 
-	tick := time.NewTicker(c.PollInterval)
+	tick := time.NewTicker(time.Second)
 	defer tick.Stop()
 
-	timer := time.NewTimer(c.WaitDuration)
+	timer := time.NewTimer(30 * time.Second)
```
Comment (fyrchik) [inline cmd/neofs-adm/internal/modules/morph/initialize.go:340]: Could you use a (local) constant here with a link to #1741 ?

File: cmd/neofs-adm/internal/modules/morph/n3client.go [hunk 2]
```diff
@@ -47,10 +49,10 @@ type Client interface {
 }
 
 type clientContext struct {
-	Client       Client
-	Hashes       []util.Uint256
-	WaitDuration time.Duration
-	PollInterval time.Duration
+	Client          Client
+	CommitteeAct    *actor.Actor
```
Comment (fyrchik) [inline cmd/neofs-adm/internal/modules/morph/n3client.go:53]: We need a short comment here, at least about the `Global` scope.

File: cmd/neofs-adm/internal/modules/morph/initialize_transfer.go [hunk 3]
```diff
@@ -129,7 +129,12 @@ func (c *initializeContext) multiSign(tx *transaction.Transaction, accType strin
 	if err != nil {
 		return fmt.Errorf("incomplete signature: %w", err)
 	}
-	tx.Scripts = append(tx.Scripts, *w)
+
+	if len(tx.Scripts) == 0 {
+		tx.Scripts = append(tx.Scripts, *w)
+	} else {
+		tx.Scripts[0] = *w
+	}
```
[the remark this answers is not in the document]
Comment (carpawell) [reply cmd/neofs-adm/internal/modules/morph/initialize_transfer.go:137]: the problem appears in the `sendCommitteeTx` when we prepare the TX with the new `MakeUnsignedRun` (the original [reason](https://github.com/nspcc-dev/neofs-node/pull/1705#discussion_r949013572) for that PR). returned TX already contains non-empty `Scripts` and that (old) code breaks signing

could be easily reproduced in `TestInitialize` without that change

i assume there are better solutions, if you have one, please, help me here

File: cmd/neofs-adm/internal/modules/morph/initialize.go (hunk 1 above)
Comment (carpawell) [reply cmd/neofs-adm/internal/modules/morph/initialize.go:340]: done

File: cmd/neofs-adm/internal/modules/morph/n3client.go (hunk 2 above)
Comment (carpawell) [reply cmd/neofs-adm/internal/modules/morph/n3client.go:53]: added comments to all the "client" fields

File: cmd/neofs-adm/internal/modules/morph/initialize_transfer.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (fyrchik) [reply cmd/neofs-adm/internal/modules/morph/initialize_transfer.go:137]: I am not ready to say now, I guess this needs some research. I am concerned about the order of `Scripts`, but it seems we always have committee as the first signer, so your approach should be ok.

**Pull request merged**